Fix grammar in documentation (#210)
vippsas/vipps-magento: docs/FAQ.md

## How do I refund an order?

> For orders refunding Adobe Commerce proposes [Adobe Commerce: Credit Memo](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/order-management/credit-memos/credit-memos) functionality.

Author: ```suggestion
For orders refunding Adobe Commerce, we propose [Adobe Commerce: Credit Memo](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/order-management/credit-memos/credit-memos) functionality.
```

## Why are some orders missing in Adobe Commerce?

> This scenario is possible for express payment flow. Unlike a regular payment, order is not created before redirecting to

Author: ```suggestion
It is possible for some orders to be missing when using the Express payment flow because, unlike a regular payment, the order is not created before redirecting to
```

> Vipps Landing page. If the transaction was successfully initiated by client, a new record

Author: ```suggestion
Vipps MobilePay Landing page. If the transaction was successfully initiated by client, a new record
```
